docs/release_notes.md

```diff
@@ -22,6 +22,7 @@
   - `osm_proxy_xds_request_count`
 - Support for Kubernetes 1.20 and 1.21 has been dropped
 - Multi-arch installation supported by the Chart Helm by customizing the `affinity` and `nodeSelector` fields
+- Root service in a `TrafficSplit` configuration must have a selector matching the pods backing the leaf services. The legacy behavior where a root service without a selector matching the pods backing the leaf services is able to split traffic, has been removed.
```

Comment:
Maybe mark this change as breaking?

Reply:
It's already listed under the `Breaking changes` section.